ystia/yorc
# Bugfix/gh 324 add ctx props to events

# Pull Request description

## Description of the change

### What I did
1. Use the context to build event information for all events
2. Emit event on status change when related task status is updated
3. Add more information on operation and node name in case of custom command
4. add this information in events CLI

### How to verify it
Deploy the python component and execute custom commands say_hello/say_goofbye
(https://github.com/ystia/yorc-a4c-plugin/tree/develop/tosca-samples/org/ystia/yorc/samples/python)
Check with CLI (yorc d events -b) that operation name and node are present in the custom command related events

### Description for the changelog
* Custom command events doesn't provide enough information ([GH-324](https://github.com/ystia/yorc/issues/324))
## Applicable Issues
fixes #324
